src/app/core/data/request.models.ts

```diff
@@ -217,6 +220,46 @@ export class IntegrationRequest extends GetRequest {
   }
 }
 
+export class CreateMetadataSchemaRequest extends PostRequest {
```

Comment:
A basic description or Typedocs should be added for all these new `*Request` classes in this file.  (See below as well)

Reply:
Added